src/components/plots/areaPlot.ts

```diff
@@ -26,13 +26,32 @@ export module Plots {
       this.attr("stroke", defaultColor);
     }
 
+    public y(): Plots.AccessorScaleBinding<number, number>;
+    public y(y: number | Accessor<number>): Area<X>;
+    public y(y: number | Accessor<number>, yScale: QuantitativeScale<number>): Area<X>;
+    public y(y?: number | Accessor<number>, yScale?: QuantitativeScale<number>): any {
+      if (y == null) {
+        return super.y();
+      }
+      super.y(y, yScale);
+
+      if (yScale != null) {
+        var y0Binding = this.y0();
+        var y0 = y0Binding && y0Binding.accessor;
```

Comment:
y0 is always predefined to be at least to be 0 so y0 can be assumed to be nonnull i believe